src/hal/src/image.rs

```diff
@@ -589,7 +589,14 @@ pub struct SubresourceLayers {
     pub layers: Range<Layer>,
 }
 
-/// A subset of resources contained within an image.
+/// When creating an image view, used for describing the view's purpose and which part
+/// of the image should be accessed. Examples of purpose would be if the view should be
+/// used as a color, depth, or stencil target (or any combination). A range of mipmap levels
```

Comment:
Yes, there is an aspect flag but it's not really indicating the usage purpose but rather with 'plane' of the image will be part of the subresource.

Reply:
Ok! The wording "purpose" came from the [Vulkan tutorial](https://vulkan-tutorial.com/Drawing_a_triangle/Presentation/Image_views). So the aspect flag is for 'filtering' which aspect of the image will be a part of the subresource?